DataConnectors/ForgeRock_CEF.json

```diff
@@ -0,0 +1,112 @@
+    },
+    {
+      "description" : "Autocluster logins by IP addresses ",
+      "query" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\") | where DeviceProduct in (\"Trust Partner Network\")\n| where DeviceAction =~ \"FAILED\"\n| project SourceIP \n| evaluate autocluster(0.6)"
+    }
+  ],
+  "dataTypes" : [
+    {
+      "name" : "CommonSecurityLog (ForgeRock)",
+      "lastDataReceivedQuery" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\") | where DeviceProduct == \"Trust Partner Network\"\n| summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+    }
+  ],
+  "connectivityCriterias" : [
+    {
+      "type" : "IsConnectedQuery",
+      "value" : [
+        "\nCommonSecurityLog\n| where DeviceVendor == \"PROVIDER NAME\"\n| where DeviceProduct == \"APPLIANCE NAME\"\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+      ]
+    }
+  ],
+  "availability" : {
+    "status" : 1
+  },
+  "permissions" : {
+    "resourceProvider" : [
+      {
+        "provider" : "Microsoft.OperationalInsights/workspaces",
+        "permissionsDisplayText" : "read and write permissions are required.",
+        "providerDisplayName" : "Workspace",
+        "scope" : "Workspace",
+        "requiredPermissions" : {
+          "read" : true,
+          "write" : true,
+          "delete" : true
+        }
+      }
+    ]
+  },
+  "instructionSteps" : [
+    {
+      "title" : "Configuration for the ForgeRock Common Audit (CAUD) for Microsoft Azure Sentinel",
+      "description" : "In ForgeRock, install and configure this Common Audit (CAUD) for Azure Sentinel per the documentation at https://github.com/javaservlets/SentinelAuditEventHandler. Next, in Azure, follow the below CEF steps."
+    },
+    {
+      "title" : "\n\n\n1. Linux Syslog agent configuration",
+      "description" : "Install and configure the Linux agent to collect your Common Event Format (CEF) Syslog messages and forward them to Azure Sentinel.\n\n> Notice that the data from all regions will be stored in the selected workspace",
+      "innerSteps" : [
+        {
+          "title" : "1.1 Select or create a Linux machine",
+          "description" : "Select or create a Linux machine that Azure Sentinel will use as the proxy between your security solution and Azure Sentinel this machine can be on your on-prem environment, Azure or other clouds."
+        },
+        {
+          "title" : "1.2 Install the CEF collector on the Linux machine",
+          "description" : "Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector colects CEF messages on port 514 TCP.\n\n> 1. Make sure that you have Python on your machine using the following command: python –version.\n\n> 2. You must have elevated permissions (sudo) on your machine.",
```

Comment:
Please fix the typo - collects (it currently shows as colects).

Reply:
done